README.arm.md

```diff
@@ -129,3 +129,43 @@ finding appropriate `crouton` help.
 
 On the current [Scaleway](http://scaleway.com) ARM servers, the Julia
 build works out of the box.
+
+## nVidia Jetson TX2
+
+Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html) platform with minimal configuration changes. A full multi-threaded build, including LLVM, will complete in around two hours. All tests pass and CUDA functionality is available through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl). 
+
+Starting from the default configuration flashed by [Jetpack 3.0](https://developer.nvidia.com/embedded/jetpack):
+
+```
+sudo apt-get install libssl-dev
+```
+
+### Julia 0.5.1
```

Comment:
Also, both of the PR's (https://github.com/JuliaLang/julia/pull/18100 and https://github.com/JuliaLang/julia/pull/19022) that fixes generic build are already marked for backport. And IIUC @staticfloat have already tested / needed them to build the generic ARM binary on the buildog. I've also just marked https://github.com/JuliaLang/julia/pull/21447 as backport pending 0.5 since that should make it only necessary to set `MARCH=armv8-a` to work around openlibm bug (fixed by https://github.com/JuliaLang/openlibm/pull/151).